### PR TITLE
Support ASIC/SDK health event

### PR DESCRIPTION
**Pull request by stephenxs**
New table definition for ASIC/SDK health event

### CODE REVIEW

**Comment by stephenxs** [comment]
Hi @prsunny 
Could you please approve and merge this PR? It is for ASIC/SDK health event and contains only table name definition.
All other PRs of the feature depend on it.
Thanks

**Review by prsunny — APPROVED** [review]

File: common/schema.h [hunk 1]
```diff
@@ -521,6 +523,8 @@ namespace swss {
 
 #define STATE_TRANSCEIVER_INFO_TABLE_NAME           "TRANSCEIVER_INFO"
 
+#define ASIC_SDK_HEALTH_EVENT_TABLE_NAME            "ASIC_SDK_HEALTH_EVENT_TABLE"
```
Comment (prsunny) [inline common/schema.h:526]: missed this while reviewing, could you please prefix this with STATE_ ? (please raise a separate PR)

Comment (stephenxs) [reply common/schema.h:526]: Sure.

Comment (stephenxs) [reply common/schema.h:526]: https://github.com/sonic-net/sonic-swss-common/pull/867

**Pull request merged**